### DISCUSSION TITLE
Laravel Shortcodes

### DISCUSSION THREAD

**Discussion by tehwave** [HassanZahirnia/laravel-package-ocean#133] [Package Suggestions]
Hi there,

My package [Laravel Shortcodes](https://github.com/tehwave/laravel-shortcodes) implements the [WordPress Shortcode API](https://codex.wordpress.org/Shortcode_API) in a modern, object-oriented way that feels native to Laravel.

```php
use tehwave\Shortcodes\Shortcode;

$compiledContent = Shortcode::compile('[uppercase]Laravel Shortcodes[/uppercase]');

// LARAVEL SHORTCODES
```

Laravel Shortcodes has a wide range of supported versions (min. is Laravel 6 and PHP 7.2) and with (almost) full test coverage.

Please consider including this package as I believe it'd be useful for developers looking to implement page builders and community-authored content on their websites.

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!
